src/loader-components/_ExampleTemplate.jsx

```diff
@@ -33,51 +38,129 @@ const defaultProps = {
   isExpanded: false,
 };
 
+/**
+ * Enables focus styles for the target of the given event. Typically used as an onBlur callback on selectable elements.
+ */
+const enableFocusStyles = (event) => {
+  event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+};
+
+/**
+ * Disables focus styles for the target of the given event. Typically used as an onMouseDown callback on selectable elements.
+ */
+const disableFocusStyles = (event) => {
+  event.currentTarget.setAttribute('data-focus-styles-enabled', 'false');
+};
+
 const ExampleTemplate = ({
-  example, exampleSrc, title, description, isExpanded,
+  example, exampleSrc, exampleCssSrc, title, description, isExpanded,
 }) => {
+  let isCssExpanded;
   const [codeIsVisible, setCodeIsVisible] = useState(isExpanded);
-  const [isBackgroundTransparent, setIsBackgroundTransparent] = useState(false);
+  const [cssIsVisible, setCssIsVisible] = useState(isCssExpanded);
```

Comment:
you can remove the isCssExpanded var.

```suggestion
  const [cssIsVisible, setCssIsVisible] = useState(false);
```

Reply:
Updated [here](https://github.com/cerner/terra-dev-site/pull/281/commits/423db62d308b80ea7bb01ea4b9dc50a39d434cb7).